app/src/main/java/com/dkanada/gramophone/adapter/song/ShuffleButtonSongAdapter.java

```diff
@@ -65,6 +67,10 @@ public void onBindViewHolder(@NonNull final SongAdapter.ViewHolder holder, int p
         }
     }
 
+    public void setFavorite(boolean favorite) {
+        onlyFavorites = favorite;
```

Comment:
Can we match the variable name with the setter name? Either `favorite` on the private variable or `favorites` for both sounds fine.